2022-RapidReact/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +38,9 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+    kStart.toggleWhenPressed(new ActivateConveyor(storageSystem, Constants.STORAGE_SPEED));
```

Comment:
tem certeza que essa é a melhor opção?toggleWhenPressed


toggleWhenActive/toggleWhenPressed
This binding toggles a command, scheduling it when a trigger changes from inactive to active (or a button is initially pressed), and canceling it under the same condition if the command is currently running. Note that while this functionality is supported, **_toggles are not a highly-recommended option for user control_**, as they require the driver to keep track of the robot state. The preferred method is to use two buttons; one to turn on and another to turn off. Using a StartEndCommand or a ConditionalCommand is a good way to specify the commands that you want to be want to be toggled between.

https://docs.wpilib.org/pt/stable/docs/software/commandbased/binding-commands-to-triggers.html#togglewhenactive-togglewhenpressed